docs/src/dev/future/proposal-multi-threaded-transaction-5.asciidoc

```diff
@@ -0,0 +1,146 @@
+////
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+////
+image::apache-tinkerpop-logo.png[width=500,link="https://tinkerpop.apache.org"]
+
+*x.y.z - Proposal 4*
+
+Proposal for adding support for tx which are not thread bound.
```

Comment:
Based on the rest of the document, this seems to be a design of how to modify TinkerTransactionGraph. Could you expand on how the transaction model itself changes? This information is more helpful for providers to determine if the new model fits with their systems. They may not know how TinkerTransactionGraph works.